src/Symfony/Component/Ldap/Adapter/EntryManagerInterface.php

```diff
@@ -21,6 +21,9 @@
  * @author Charles Sarrazin <charles@sarraz.in>
  * @author Bob van de Vijver <bobvandevijver@hotmail.com>
  * @author Kevin Schuurmans <kevin.schuurmans@freshheads.com>
+ *
+ * @method void addAttributeValues(Entry $entry, string $attribute, array $values)
+ * @method void removeAttributeValues(Entry $entry, string $attribute, array $values)
```

Comment:
to be able to declare a `void` return type in SF5, we must mark `EntryManager` as `@final since Symfony 4.3`

Reply:
added